exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -2323,44 +2295,53 @@ func (e *Exchange) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item)
 			if a == asset.Margin && !symbols[x].IsMarginEnabled {
 				continue
 			}
-			pair, enabled, err := e.MatchSymbolCheckEnabled(symbols[x].Symbol, a, true)
-			if err != nil && !errors.Is(err, currency.ErrPairNotFound) {
-				return err
-			}
-			if !enabled {
-				continue
-			}
 			l = append(l, limits.MinMaxLevel{
-				Key:                     key.NewExchangeAssetPair(e.Name, a, pair),
-				AmountStepIncrementSize: symbols[x].BaseIncrement,
-				QuoteStepIncrementSize:  symbols[x].QuoteIncrement,
-				PriceStepIncrementSize:  symbols[x].PriceIncrement,
-				MinimumBaseAmount:       symbols[x].BaseMinSize,
-				MaximumBaseAmount:       symbols[x].BaseMaxSize,
-				MinimumQuoteAmount:      symbols[x].QuoteMinSize,
-				MaximumQuoteAmount:      symbols[x].QuoteMaxSize,
+				Key:                     key.NewExchangeAssetPair(e.Name, a, symbols[x].Symbol),
+				AmountStepIncrementSize: symbols[x].BaseIncrement.Float64(),
+				QuoteStepIncrementSize:  symbols[x].QuoteIncrement.Float64(),
+				PriceStepIncrementSize:  symbols[x].PriceIncrement.Float64(),
+				MinimumBaseAmount:       symbols[x].BaseMinSize.Float64(),
+				MaximumBaseAmount:       symbols[x].BaseMaxSize.Float64(),
+				MinimumQuoteAmount:      symbols[x].QuoteMinSize.Float64(),
+				MaximumQuoteAmount:      symbols[x].QuoteMaxSize.Float64(),
+				Listed:                  symbols[x].TradingStartTime.Time(),
 			})
 		}
 	case asset.Futures:
 		contract, err := e.GetFuturesOpenContracts(ctx)
 		if err != nil {
 			return err
 		}
+
 		l = make([]limits.MinMaxLevel, 0, len(contract))
 		for x := range contract {
-			pair, enabled, err := e.MatchSymbolCheckEnabled(contract[x].Symbol, a, false)
-			if err != nil && !errors.Is(err, currency.ErrPairNotFound) {
+			pair, err := e.MatchSymbolWithAvailablePairs(contract[x].Symbol, a, false)
+			if err != nil {
 				return err
 			}
-			if !enabled {
-				continue
+
+			priceDivisor := 1.0
+			if contract[x].Symbol[:2] == "10" { // handle 1000SHIBUSDT, 1000PEPEUSDT etc; screen 1INCHUSDT
```

Comment:
```suggestion
			if contract[x].Symbol[:2] == "10" { // handle 1000SHIBUSDT, 1000PEPEUSDT etc; exclude 1INCHUSDT
```
ehhhhh I'm not fussed